src/ORM/Behavior/Translate/ShadowTableStrategy.php

```diff
@@ -483,6 +483,7 @@ protected function rowMapper($results, $locale)
             unset($row['translation']);
 
             if ($hydrated) {
+                /** @psalm-suppress PossiblyInvalidMethodCall */
                 $row->clean();
```

Comment:
Resetting $row type with @var didn't work here (it is set above). This code does a lot of type juggling that might be written simpler.